DigitalLearningSolutions.Data/Models/Courses/DelegateCourseDetails.cs

```diff
@@ -13,11 +14,14 @@ public class DelegateCourseDetails
         {
             DelegateCourseInfo = delegateCourseInfo;
             CustomPrompts = customPrompts;
-            AttemptStats = attemptStats;
+            var passRate = attemptStats.totalAttempts == 0
+                ? 0
+                : Math.Round(100 * attemptStats.attemptsPassed / (double)attemptStats.totalAttempts);
+            AttemptStats = (attemptStats.totalAttempts, attemptStats.attemptsPassed, passRate);
         }
 
         public DelegateCourseInfo DelegateCourseInfo { get; set; }
         public List<CustomPromptWithAnswer> CustomPrompts { get; set; }
-        public (int totalAttempts, int attemptsPassed) AttemptStats { get; set; }
+        public (int totalAttempts, int attemptsPassed, double passRate) AttemptStats { get; set; }

```

Comment:
Is there a reason for this not to be a class (or struct) of its own? Then it could hold the calculation which you've added in the constructor.